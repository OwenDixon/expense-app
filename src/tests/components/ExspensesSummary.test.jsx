import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from "../../components/ExpensesSummary.jsx";

test('Should correctly render ExpensesSummary with 1 expense', () => {

    const wrapper = shallow(<ExpensesSummary expenseCount={1} expenseTotal={345} />)
    expect(wrapper).toMatchSnapshot();

});

test('Should correctly render ExpensesSummary with multiple expense', () => {
   
    const wrapper = shallow(<ExpensesSummary expenseCount={12} expenseTotal={343325} />)
    expect(wrapper).toMatchSnapshot();

});