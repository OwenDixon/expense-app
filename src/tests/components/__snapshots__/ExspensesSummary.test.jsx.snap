// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $3.45
  </h1>
</div>
`;

exports[`Should correctly render ExpensesSummary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    12
     
    expenses
     totalling 
    $3,433.25
  </h1>
</div>
`;
